test/requirements.txt

```diff
@@ -10,3 +10,4 @@ pylint==1.4.1
 pep8==1.5.7
 pep257
 freezegun
+-e git+https://github.com/Alignak-monitoring/mod-example.git#egg=mod-example
```

Comment:
Bad: commit message says it removes irrelevant files but obviously in fact it does more than that !!

also in test_external_mapping + test_module_as_package + test_modulemanager

Please don't do that !!